src/containers/EditListingPage/EditListingWizard/EditListingPricingAndStockPanel/EditListingPricingAndStockPanel.js

```diff
@@ -76,14 +78,24 @@ const EditListingPricingAndStockPanel = props => {
   const publicData = listing?.attributes?.publicData;
   const unitType = publicData.unitType;
   const listingTypeConfig = getListingTypeConfig(publicData, listingTypes);
+  const transactionProcessAlias = listingTypeConfig.transactionType.alias;
 
   const hasInfiniteStock = STOCK_INFINITE_ITEMS.includes(listingTypeConfig?.stockType);
 
   const isPublished = listing?.id && listing?.attributes?.state !== LISTING_STATE_DRAFT;
-  const priceCurrencyValid =
-    marketplaceCurrency && initialValues.price instanceof Money
-      ? initialValues.price?.currency === marketplaceCurrency
-      : !!marketplaceCurrency;
+
+  // Don't render the form if the assigned currency is different from the marketplace currency
```

Comment:
You added the fix to PricingAndStock panel, but there's also Pricing panel, which is used with booking and inquiries. I guess that fix should be applied there too (but you need to take default-inquiry process into account)